jme3-core/src/com/jme3/gde/core/scene/SceneApplication.java

```diff
@@ -715,12 +712,12 @@ public static void showStartupErrorMessage(Throwable exception) {
         NotifyUtil.show("Error starting OpenGL context!", "Click here to go to troubleshooting web page.", MessageType.EXCEPTION, lst, 0);
         logger.log(Level.INFO, exception.getMessage(), exception);
```

Comment:
Actually this could also be changed into ExceptionUtils.show(??) for the user to have a nice exception msg